components/JsonDisplayCollapse.tsx

```diff
@@ -0,0 +1,18 @@
+export function JsonDisplayCollapse({
```

Comment:
Since you are already exporting at the bottom of each of your components, export is not needed here!
```suggestion
function JsonDisplayCollapse({
```

Reply:
Done!

Reply:
💯 awesome work - side note, meant to imply this for all your components, if you feel like cleaning up all those superfluous `export`'s at the top of each of 'em 😄 not very important, just FYI